boltkit/controller.py

```diff
@@ -225,7 +225,7 @@ def user_record(user, password):
     m = sha256()
     m.update(salt)
     m.update(bstr(password))
-    return bstr("%s:SHA-256,%s,%s:" % (user, hex_bytes(m.digest()), hex_bytes(salt)))
+    return bstr(user) + b":SHA-256," + bstr(hex_bytes(m.digest())) + b"," + bstr(hex_bytes(salt)) + b":"
```

Comment:
I think `hex_bytes` should always output `bytes` so the `bstr` conversion is redundant.

Reply:
Should be addressed now, still building my python knowledge :)